build/scripts/write_image_digests.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+
+readarray -d '' devfiles < <(find "$1" -name 'devfile.yaml' -print0)
+for image in $(yq -r '.components[]?.image' "${devfiles[@]}" | grep -v "null" | sort | uniq); do
+  echo "Rewriting image $image"
+  digest=$(skopeo inspect "docker://${image}" | jq -r '.Digest')
```

Comment:
This works with one caveat: the sha256sum for the image pushed to Quay is not the same as the image pushed to RHCC.

Reply:
Wouldn't the build take place with the images set as desired? I didn't realize we were building the registry and then updating the images inside it.

Reply:
I assume you would run this build script for both the registry image destinated to quay, and the registry image destinated to rhcc anyway. So it should not be a problem.

Reply:
> and then updating the images inside it.

So this change means that if we have to rebuild any of the stacks-* or plugin-* containers used in the plugin and devfile registries, we have to ALSO then rebuild the registries to pull in the new SHAs. Yay for extra churn. :)

> run this build script for both 

yes, [CRW-214](https://issues.redhat.com/browse/CRW-214) will solve all our problems here :D